nano/core_test/active_transactions.cpp

```diff
@@ -46,6 +46,10 @@ TEST (active_transactions, confirm_election_by_request)
 	// Ensure election on node1 is already confirmed before connecting with node2
 	ASSERT_TIMELY (5s, nano::test::confirmed (node1, { send1 }));
 
+	// Wait for the election to be removed and give time for any in-flight vote broadcasts to settle
+	ASSERT_TIMELY (5s, node1.active.empty ());
+	WAIT (250ms);
```

Comment:
This seems like a hack. On a very slow computer, 250 ms might not be enough for the network to settle.
Ideally, we want something more concrete.

Reply:
I would use a higher wait period, personally I always use 1 second. While tracking those generated votes through vote processor, network stack etc could be correct, it's not practical or worth spending time on. A longer wait time here should be fine.